src/traces/scattercarpet/style.js

```diff
@@ -0,0 +1,27 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var scatterStyle = require('../scatter/style');
+
+
+module.exports = function style(gd) {
```

Comment:
I've included it and restored the same check that scatterternary uses to either run it or take for granted that the scatter trace has already run it.